businesscentral-monitoring/tests/features/businesscentral.feature

```diff
@@ -1,10 +1,12 @@
+@jboss-bpmsuite-7
```

Comment:
What is the difference between using just `@jboss-bpmsuite-7` and `@jboss-bpmsuite-7/bpmsuite70-businesscentral`? I have been using the full name. Does it mean it is not necessary?

Reply:
if we tag the test only with the prefix image name, in this case jboss-bpmsuite-7, all other images with have the same prefix will run these tests.
But if it is specific to a single image the full name can be used.

Reply:
Cool, thanks for the explanation!